docker-definition/fixtures/lms/users.json

```diff
@@ -1,92 +1,83 @@
 [
-{
-  "model": "auth.user",
-  "pk": 1,
-  "fields": {
-    "password": "!h7gSgJg8iHJKxD9oJPNZWpRtgwK4yRazecqj9jSV",
-    "last_login": null,
-    "is_superuser": false,
-    "username": "ecommerce_worker",
-    "first_name": "",
-    "last_name": "",
-    "email": "ecommerce_worker@fake.email",
-    "is_staff": false,
-    "is_active": true,
-    "date_joined": "2019-03-29T17:52:56.557Z",
-    "groups": [],
-    "user_permissions": []
+  {
+    "model": "auth.user",
+    "pk": 1,
+    "fields": {
+      "password": "pbkdf2_sha256$36000$Rsbs2MAwwD4X$BWVn4zJOt9kO5u2ihgVeIQMnjrtpDzpclX8F2r1pbTw=",
+      "last_login": null,
+      "is_superuser": false,
+      "username": "user",
+      "first_name": "",
+      "last_name": "",
+      "email": "user@example.com",
+      "is_staff": false,
+      "is_active": true,
+      "date_joined": "2019-03-29T20:47:03.191Z",
+      "groups": [],
+      "user_permissions": []
+    }
+  },
+  {
+    "model": "student.userprofile",
+    "pk": 1,
+    "fields": {
+      "user": 1,
+      "name": "Student User",
+      "year_of_birth": 1920
+    }
+  },
+  {
+    "model": "auth.user",
+    "pk": 2,
+    "fields": {
+      "password": "pbkdf2_sha256$36000$VAx8S2zQhKks$MtTu2pjY0CQegB+Q5O3DkN8ncw6Hk8O21KGJNY5+ADU=",
+      "last_login": null,
+      "is_superuser": false,
+      "username": "staff",
+      "first_name": "",
+      "last_name": "",
+      "email": "staff@example.com",
+      "is_staff": true,
+      "is_active": true,
+      "date_joined": "2019-03-29T20:47:20.839Z",
+      "groups": [],
+      "user_permissions": []
+    }
+  },
+  {
+    "model": "student.userprofile",
+    "pk": 2,
+    "fields": {
+      "user": 2,
+      "name": "Staff User",
+      "year_of_birth": 1920
+    }
+  },
+  {
+    "model": "auth.user",
+    "pk": 3,
+    "fields": {
+      "password": "pbkdf2_sha256$36000$TGGr1rmjd1eG$kjM3FkV/Y/LLFzzbps0HFWIq7gYW5h3z7Bx3+TixIJA=",
+      "last_login": null,
+      "is_superuser": true,
+      "username": "superuser",
+      "first_name": "",
+      "last_name": "",
+      "email": "superuser@example.com",
+      "is_staff": true,
+      "is_active": true,
+      "date_joined": "2019-03-29T20:47:40.165Z",
+      "groups": [],
+      "user_permissions": []
+    }
+  },
+  {
+    "model": "student.userprofile",
+    "pk": 3,
+    "fields": {
+      "user": 3,
+      "name": "Admin User",
+      "year_of_birth": 1920
+    }
   }
-},
-{
-  "model": "auth.user",
-  "pk": 2,
-  "fields": {
-    "password": "!KSOwh838JwnDre2DnPgl4nAAkY0WPPvb4PnbYKvS",
-    "last_login": null,
-    "is_superuser": false,
-    "username": "login_service_user",
```

Comment:
AFAIK the `login_service_user` is needed for the login workflow:
https://github.com/edx/edx-platform/blob/open-release/ironwood/openedx/core/djangoapps/user_authn/migrations/0001_data__add_login_service.py#L14
https://github.com/edx/edx-platform/blob/5d62456250d9f16aa1bb379980f1fb8782ca7b27/lms/envs/common.py#L3477

Reply:
True. Good catch.

The migration we run to generate the dump have this effect:
```LOCK TABLES `auth_user` WRITE;
/*!40000 ALTER TABLE `auth_user` DISABLE KEYS */;
INSERT INTO `auth_user` VALUES (1,'!u3UJLCFFjwzbsCdb3UAhaZgLwhRVZJwROJ8fmnHt',NULL,0,'ecommerce_worker','','','ecommerce_worker@fake.email',0,1,'2020-04-10 19:02:45.421053'),(2,'!hsVwRfW
jhvC4aul4212HjgNiye3F30H4kYtMOV6B',NULL,0,'login_service_user','','','login_service_user@fake.email',0,1,'2020-04-10 19:05:25.238396');
/*!40000 ALTER TABLE `auth_user` ENABLE KEYS */;
UNLOCK TABLES;
``` 
(that SQL snippet is taken from one of our images, inside `/openedx/empty_dump.sql.bz2`)

So I still think we should remove that user from our fixtures, but we also need to avoid using user id `1`, as the primary key is already taken by the only user created by the migrations.